app/src/main/java/com/example/campusguide/calendar/Calendar.kt

```diff
@@ -31,7 +35,6 @@ class Calendar constructor(val activity: MapsActivity, userEmail: String) {
 
     private val email: String = userEmail
     private var calendarsList: ArrayList<Pair<Long, String>> = arrayListOf()
-    private lateinit var selectedCalendar: Pair<Long, String>
 
     @SuppressLint("MissingPermission")
```

Comment:
I know this isn't from this pull request, but what's up with this?

Reply:
An inline comment saying why we're suppressing things would be good

Reply:
> addressed in inline comment but i'll respond to you here as well: the permissions are checked elsewhere. the suppression is purely so that androidstudio will stop harassing us about it in every class that uses the permissions we've already checked for

Reply:
Okay, it'd be good to add where they're being checked, or maybe a todo for figuring out a better way to fix that.